src/main/java/frc/robot/RobotContainer.java

```diff
@@ -48,16 +51,35 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
+
+    // Configure the robot to be driven by the left stick for steering and for
+    // throttle
     drive.setDefaultCommand(
         new ArcadeDrive(drive, () -> -mainController.getLeftY(), () -> -mainController.getLeftX()));
 
+    // // Arcade drive but left stick for throttle and right stick controls steering
+    // drive.setDefaultCommand(
+    //     new ArcadeDrive(drive,
+    //         () -> -mainController.getLeftY(),
+    //         () -> -mainController.getRightX()));
+
+    // Both sticks control steering using TankDrive
+    // drive.setDefaultCommand(
+    //        new TankDrive(drive,
+    //           () -> -mainController.getLeftY(),
+    //           () -> -mainController.getRightY()));
+
     arm.setDefaultCommand(new ArmCommand(arm, () -> -mainController.getRightY()));
+
     intake.setDefaultCommand(new IntakeCommand(intake, () -> -mainController.getRightX()));
+
     shooter.setDefaultCommand(
         new ShooterCommand(
             shooter,
             () -> mainController.getLeftTriggerAxis(),
             () -> mainController.getRightTriggerAxis()));
+
+    mainController.a().toggleOnTrue(new AutonomousDistance(drive));
```

Comment:
I'd add a comment saying that this binds the "A" button to execute the AutonomousDistance routine.

Reply:
I updated this to be part of the autonomous command logic.